### PR TITLE
Workaround to avoid native crash

### LINKED ISSUE #1080: Client crash (every 3-4 games)
Since 0.9.x, client crashes on me quite often.
Game stays up, but since client crashes, I lose conn to all players.

Here is the Java crash log:
https://gist.github.com/tomaskir/c88e45eea3acdb3a17d80ae10084bbcd

### PR DESCRIPTION
**Pull request by 1-alex98**
Fixes #1080

### CODE REVIEW

**Review by micheljung — CHANGES_REQUESTED** [review]
Something's wrong with the changes in this PR

**Comment by 1-alex98** [comment]
sorry @micheljung i rebased onto reinitiate-conectivity code to be able to build a test release? But forgot to close PR or make a different version for PR... I am waiting for the feedback of @tomaskir

File: src/main/java/com/faforever/client/chat/ChannelTabController.java [hunk 1]
```diff
@@ -364,8 +364,8 @@ private void removeUserMessageClass(ChatChannelUser chatUser, String cssClass) {
     if (cssClass.isEmpty()) {
       return;
     }
-    Platform.runLater(() -> getJsObject().call("removeUserMessageClass", String.format(USER_CSS_CLASS_FORMAT, chatUser.getUsername()), cssClass));
-
+    //Workaround for issue #1080 https://github.com/FAForever/downlords-faf-client/issues/1080
+    Platform.runLater(() -> engine.executeScript("removeUserMessageClass(\'" + String.format(USER_CSS_CLASS_FORMAT, chatUser.getUsername()) + "\',\'" + cssClass + "\');"));
```
Comment (1-alex98) [inline src/main/java/com/faforever/client/chat/ChannelTabController.java:368]: Only line that changed

**Comment by 1-alex98** [comment]
Downlords update of the version did not succeed

**Review by Brutus5000 — APPROVED** [review]

**Pull request merged**